pinchy/src/format_helpers.rs

```diff
@@ -31,6 +31,14 @@ pub mod fs_constants {
     pub const FSPICK_SYMLINK_NOFOLLOW: u32 = 0x00000002;
     pub const FSPICK_NO_AUTOMOUNT: u32 = 0x00000004;
     pub const FSPICK_EMPTY_PATH: u32 = 0x00000008;
+
+    // openat2() resolve flags
+    pub const RESOLVE_BENEATH: u64 = 0x1;
+    pub const RESOLVE_IN_ROOT: u64 = 0x2;
+    pub const RESOLVE_NO_MAGICLINKS: u64 = 0x4;
+    pub const RESOLVE_NO_SYMLINKS: u64 = 0x8;
+    pub const RESOLVE_NO_XDEV: u64 = 0x10;
+    pub const RESOLVE_CACHED: u64 = 0x20;
```

Comment:
These constants do not need to be declared here, libc has all of them, as you can see here:

https://docs.rs/libc/latest/libc/constant.RESOLVE_BENEATH.html?search=RESOLVE_

We don't use them on the eBPF side, so no need to declare them ourselves, just use the libc:: constants where needed.